### ISSUE TITLE
Quick Question Regarding Training and Evaluation Details

### ISSUE THREAD

**Issue by huahuakoqw** [AIR-hl/MWPO#2]
Hi Shiyue Xu,
I just wanted to say thank you for your amazing work on this project!

In the description, it mentions generating responses for all datasets using vllm (Kwon et al., 2023), with token sampling at different temperatures: 0.7 for AlpacaEval and 0.5 for the Ultrafeedback testset.

What does "0.5 for the Ultrafeedback testset" specifically refer to? The Ultrafeedback dataset is divided into the training set train_prefs-00000-of-00001.parquet and the test set test_prefs-00000-of-00001.parquet, where one is used to train the model and the other for validation. Why is the test dataset used when generating responses during inference? Shouldn't the inference use AlpacaEval2's own dataset instead?

Thank you so much for your help!

**Comment by AIR-hl** [comment]
Hi! First of all, please forgive me for not really understanding ur question. The difference in temperature settings is for the experiment shown in Figure 3. I counted the distribution of rewards on ultrafeedback and alpacaeval data respectively. Perhaps you could describe your question more clearly ～

**Comment by huahuakoqw** [comment]
<img width="277" height="83" alt="Image" src="https://github.com/user-attachments/assets/ae8ace4f-5c09-46df-9d19-82cd4ba92203" />  sorry, In Appendix B, Training and Evaluation Details, as shown in the image, highlighted in bright yellow, I don't understand why the Ultrafeedback dataset is used for model inference. Isn't it common to use the AlpacaEval2 dataset for inference?

**Comment by AIR-hl** [comment]
@huahuakoqw Got it.  AlapcaEval is indeed a common benchmark, and I have conducted detailed experiments on AlapcaEval in the **main experiment** part yet. The ultrafeedback testset result provides an auxiliary perspective as ID data compare to Alpacaeval OOD data. Overall, it's not necessary results.
